contributing_to_docs/doc_guidelines.adoc

```diff
@@ -693,36 +703,31 @@ To provide an example URL path that you do not want to render as a hyperlink, us
 ....
 
 === Internal cross-references
-Whenever possible the link to another assembly should be part of the actual sentence.
-Avoid creating links as a separate sentence that begins with "See [this assembly] for more information on x".
 
-[NOTE]
-====
-Use the relative file path (from the file you are editing, to the file you are linking to),
-even if you are linking to the same directory that you are writing in. This makes search and replace
-operations to fix broken links much easier.
+Use the relative file path (from the file you are editing to the file you are linking to), even if you are linking to the same directory that you are writing in. This makes search and replace operations to fix broken links much easier.
 
-For example, if you are writing in *_architecture/core_concepts/deployments.adoc_* and you want to
-link to *_architecture/core_concepts/routes.adoc_* then you must include the path back to the first
-level of the assembly directory:
+For example, if you are writing in *_architecture/core_concepts/deployments.adoc_* and you want to link to *_architecture/core_concepts/routes.adoc_*, then you must include the path back to the first level of the assembly directory:
 
 ----
 xref:../../architecture/networking/routes.adoc#architecture-core-concepts-routes
 ----
+
+[NOTE]
+====
+In OpenShift docs, you can only use `xref` in assemblies, not modules.
 ====
 
 .Markup example of cross-referencing
 ----
-Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or
-the xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[OpenShift CLI].
+For more information, see xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[Installing the OpenShift CLI].
 
-Before you can create a domain, you must first xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[create an application].
+For more information, see xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[Creating an application].
 ----
 
 .Rendered output of cross-referencing
-> Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or the xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[OpenShift CLI].
+> For more information, see  xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[Installing the OpenShift CLI].
 >
-> Before you can create a domain, you must first xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[create an application].
+> For more information, see xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[Creating an application].
 
```

Comment:
Global comment for this section:
While the older examples set context to the cross reference, the current recommendation does not. It might help to show an example of _before_ and _after_ of the same use case.

Consider the examples in the [Prerequisites section](https://github.com/openshift/openshift-docs/blob/main/installing/installing_aws/installing-aws-user-infra.adoc#prerequisites) of the _Installing a cluster on user-provisioned infrastructure in AWS by using CloudFormation templates_.

Based on the current recommendation, I understand that they need to change to comply with the new guideline. Correct me if this understanding is incorrect.

Reply:
I don't feel that this change forbids use of the older example's style (i.e., using it within the actual sentence). It's more that the old guidelines made it seem like that was the primary way we should be doing it, and in particular the line "Avoid creating links as a separate sentence that begins with 'See [this assembly] for more information on x'" made it seem like separate sentences were bad. This "Internal cross-references" section I think should just stay more focused on the AsciiDoc / markup side of things, and leave the more general style guidelines to the IBM Style guide (as noted in the updated document intro).

FWIW, I think your Prerequisites section examples are fine as-is! I don't think we've had a team conversation really about forbidding the "actual sentence" style; if we do, we can update this section with the outcome.

Reply:
Actually, on second thought, since we _haven't_ had that aforementioned team discussion, I should still probably leave at least one example of the older style in, even if the styling isn't explicitly discussed. I've added a version of one back in as the 2nd of the two examples:

```
For more information, see xref:../dev_guide/application_lifecycle/new_app.adoc#dev-guide-new-app[Creating an application].

Rollbacks can be performed using the REST API or the xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[OpenShift CLI].
```